README.md

```diff
@@ -1 +1,77 @@
-# Hello-Microverse-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+Hello-Microverse
+
+> One paragraph statement about the project.
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages (HTML and CSS)
+- Frameworks
+- Technologies used
+
+## Live Demos
+
+[Live Demo Link](https://livedemo.com)
```

Comment:
This live demo link is broken. Kindly fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.